test/input/beamville/beam.conf

```diff
@@ -48,7 +48,17 @@ beam.agentsim.agents.vehicles.linkToGradePercentFilePath = ${beam.inputDirectory
 beam.agentsim.agents.vehicles.fuelTypesFilePath = ${beam.inputDirectory}"/beamFuelTypes.csv"
 beam.agentsim.agents.vehicles.vehicleTypesFilePath = ${beam.inputDirectory}"/vehicleTypes.csv"
 beam.agentsim.agents.vehicles.vehiclesFilePath = ${beam.inputDirectory}"/vehicles.csv"
-beam.agentsim.agents.vehicles.sharedFleets = []
+beam.agentsim.agents.vehicles.sharedFleets = [
+  {
+    name = "fixed_non_reserving_fleet_from_file"
+    managerType = "fixed_non_reserving_fleet_from_file"
+    fixed_non_reserving_fleet_from_file {
+      vehicleTypeId = "PHEV",
+      filename = "/Users/haitam/workspace/beam/test/input/beamville/vehiclesShared.csv"
```

Comment:
Please, make sure this path exist everywhere (maybe need to add it under sourcecontrol)

Reply:
my bad didn't plan to push the conf file